STAR/draft-ietf-acme-star.md

```diff
@@ -516,6 +516,72 @@ setting it to true.
 
 If the server accepts the request, it MUST reflect the key in the Order.
 
+## Computing notBefore and notAfter of STAR Certificates
+
+We define the "nominal renewal date" as the point in time when a new short-term
+certificate for a given STAR Order is due.  It is a multiple of the Order's
+recurrent-certificate-validity which starts with the issuance of the
+first short-term certificate and it is upper bounded by the Order's
+recurrent-end-date ({{fignrd}}).
+
+~~~
+    rcv    - STAR Order's recurrent-certificate-validity
+    red    - STAR Order's recurrent-end-date
+    nrd[i] - nominal renewal date of the i-th STAR certificate
+
+
+                 .-rcv-.   .-rcv-.   .-rcv-.   ._rcv_.
+                /       \ /       \ /       \ /  red  \
+    -----------o---------o---------o---------o----X-------> t
+              nrd[0]    nrd[1]    nrd[2]    nrd[3]
+~~~
+{: #fignrd title="Nominal Renewal Date"}
+
+The rules to determine the notBefore and notAfter values of the i-th STAR
+certificate are as follows:
+
+~~~
+  notBefore = nrd[i] - predating
+  notAfter = min(nrd[i] + rcv, red)
+~~~
+
+where "predating" is the max between the (optional)
+recurrent-certificate-predate (rcp) and the amount of pre-dating that the
+server needs to add to make sure that all certificates being published are
+valid at the time of publication ({{fetching-certificates}}).  The server
+pre-dating is a fraction of rcv (1/n with n>=2).
+
+~~~
+  predating = max(rcp, rcv * 1/n)
+~~~
+
+### Example
+
+Given a server that intends to publish the next cert halfway through the
+lifetime of the previous one, and a STAR Order with the following settings:
+
+~~~
+     {
+       "recurrent-start-date": "2016-01-10T00:00:00Z",
```

Comment:
Add a comment: we assume this is also when the Order request was made

Reply:
I'm not sure I understand why this would be the case?  The semantic of recurrent-start-date is "I want my short-term certs to start being issued from this date", independently of when the Order was made.

Reply:
Correct, but the default value for start-date is when the Order is made (I think we say so somewhere...) and this would simplify the example.

Reply:
if recurrent-start-date is not supplied we assume it to be the time when all authorisations have completed successfully, not when the Order is made (should be in sec. 3.1.1.).

Reply:
Yes, well, this is close enough - the rest of the example is at the granularity of days.